src/test/java/io/muehlbachler/bswe3b/CalculatorServiceTest.java

```diff
@@ -11,5 +11,18 @@ public void testDividePositiveNumberNoRemainder() {
         assertEquals(1, calc.divide(1, 1), "1/1 should be 1");
     }
 
-    // FIXME: implement more tests
+    @Test
+    public void testDividePositiveNumberWithRemainder() {
+        assertEquals(3, calc.divide(7, 2), "7/2 should be 3");
+    }
+
+    @Test
+    public void testDivideNegativeNumberNoRemainder() {
+        assertEquals(1, calc.divide(-1, -1), "-1/-1 should be 1");
+    }
+
+    @Test
+    public void testDivideNegativeNumberWithRemainder() {
+        assertEquals(3, calc.divide(-7, -2), "-7/-2 should be 3");
```

Comment:
this is not 3 if you look at the specification and the code you wrote.

Reply:
Has been resolved in https://github.com/muhlba91/fh-burgenland-bswe-3b/pull/5/commits/2f3ffcbf7cbc405a1e82e1d83b0a7bc6e1b16903